src/applications/disability-benefits/996/tests/fixtures/mocks/feature-toggles.json

```diff
@@ -13,10 +13,7 @@
     }, {
       "name": "facilitiesPpmsSuppressPharmacies",
       "value": true
-    }, {
-      "name": "facilityLocatorFeUseV1",
-      "value": true
```

Comment:
This still has backend support, since this is not our source code, I would just ask them if they are using it

Reply:
I confirmed with @bradl3yC that the benefits team is not using this.

Reply:
Actually this is mock data for our Benefits & Memorial 1 team. It's for form 0996 (Higher-Level Review) and this is file contains placeholder data, so removing that entry won't break anything.